src/west/app/project.py

```diff
@@ -952,6 +952,9 @@ def do_add_parser(self, parser_adder):
         parser.add_argument('--stats', action='store_true',
                             help='''print performance statistics for
                             update operations''')
+        parser.add_argument('--cloned', action='store_true',
+                            help='''update the projects that are already cloned
+                            in the current workspace''')
```

Comment:
Since `update_some()` is being changed to match, I think the meaning of this option is closer to "only update a project if it's already cloned locally" or so, right?

I'm thinking about the behavior of `west update --cloned foo` for an uncloned project `foo` and feeling the current help doesn't make clear how it should work in this case.